---
title: CV
---

<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <title>{{ page.title }}</title>
</head>

<body>
  <!-- Download Link -->
  <!-- <div class="bodyText" style='margin-top: 30px; margin-bottom: 20px;'>
    <a href="/assets/pdfs/cv.pdf"><h2><u>Download CV as PDF</u></h2></a>
  </div> -->

  <!-- CV in HTML -->
  <div class="bodyText">

  <!-- Summary -->
  <h1 class="subheading">Summary</h1>
  <p>
    I am currently working as a Bioinformatics Software Engineer at LanzaTech where I develop bioinformatics software and pipelines that empower scientists to engineer microorganisms that can produce useful chemicals sustainably.
  </p>
  <p>
    Python is my <i>native</i> programming language, but I've also written a decent amount of JavaScript, HTML, CSS, and Swift. I use other tools like bash, docker, git, and AWS daily. I love to learn new technologies and use those tools to address practical needs.
  </p>
  <p>
    I currently live in Skokie, IL and in my spare time I enjoy hiking, learning Japanese, building iOS apps, and strength training.
  </p>

  See below and check out my <a href={{ site.data.links.linkedin }}><u>LinkedIn</u></a> for more info.

  <!-- Skills -->
  <h1 class="subheading">Skills: What can I do?</h1>
  <h3>Software Development</h3>
  <ul>
    <li><b>üß¨ Develop Scientific Software:</b> For example, I helped develop the <a href="https://minedatabase.ci.northwestern.edu/#/home"><u>MINE Database</u></a>, to unlock new modes of scientific inquiry. I have also used backend frameworks like flask to build web APIs, one which also uses celery to schedule tasks across distributed workers in AWS.</li>
    <li><b>ü´±üèª‚Äçü´≤üèΩ Communicate Effectively, esp. with Scientists:</b> With a STEM PhD and as a former scientist myself, I can collaborate effectively with key stakeholders on science teams during the development process.</li>
    <li><b>‚úçüèª Communicate and document effectively:</b> I am always focused on documenting my code via docstrings, type annotations, descriptive variable names, etc. The joy I get from cleaning up messy code is like no other and highly motivating. I have also used tools like Sphinx to generate API documentation for python libraries.</li>
    <li><b>üìä Data visualization:</b> I love thinking about ways to visualize data to meet user requirements. I am most experienced with python libraries such as matplotlib, seaborn, and bokeh. I have also used tools like streamlit to create interactive dashboards.</li>
    <li><b>‚òÅ AWS:</b> I have practical experience with EC2, S3, and RDS and am familiar with networking concepts (VPNs, security groups, etc.). I have also used AWS CodeDeploy to deploy apps to production. I am currently working on obtaining the Solutions Architect AWS Associate Certification.</li>
    <li><b>üè≠ High-Performance Computing:</b> I have used the Slurm Workload Manager to submit jobs on supercomputing clusters. For example, I used the Quest supercomputing cluster at Northwestern University to predict MS2 spectra for millions of compounds in parallel.</li>
    <li><b>ü´ô Containerization:</b> I frequently use Docker to containerize my applications and have recently started diving deeper into docker compose.</li>
    <li><b>üêö Scripting:</b> I am very familiar with bash scripting, as it is often used in bioinformatics applications. I have also dabbled in bat scripting for Windows.</li>
    <li><b>üíæ Build and query databases:</b> I have experience building and working with MySQL (e.g. with python's sqlalchemy driver) as well as MongoDB (e.g. with pymongo driver) databases.</li>
    <li><b>üíª Contribute to frontend development:</b> I'm familiar with vanilla JavaScript as well as basic HTML and CSS and have built basic user interfaces for APIs I've developed. I built this site using Jekyll + HTML/CSS.</li>
    <li><b>üßπ Collaborate using git, GitHub, and Jira:</b> submit issues, make commits, merge changes, review and submit pull requests, use features like interactive rebase to keep commit history clean. I have used these tools for both solo projects as well as collaborative projects.</li>
    <li><b>üì± Develop iOS applications using SwiftUI:</b> As a side project, I'm currently working on my first iPhone app - a crossword game for Japanese language learners (see blog for details).</li>
  </ul>
  <h3>Scientific Background</h3>
  <ul>
    <li><b>üí≠ Design & Analysis of Experiments:</b> I absolutely love working with experimentalists to design experiments and analyses that best answer the most pressing questions. I have experience with both frequentist statistics as well as Bayesian statistical modeling.</li>
    <li><b>ü§ñ Machine Learning/AI:</b> Analyze complex datasets and make predictions. For example, I built a random forest regressor model using scikit-learn to predict chromatographic retention times in order to annotate unknown compounds in a metabolomics dataset.</li>
    <li><b>üß¨ Bioinformatics:</b> I have experience processing and analyzing large NGS, transcriptomics, proteomics, and metabolomics datasets.</li>
    <li><b>üß™ Cheminformatics:</b> I have used tools like RDKit and mordred to process, analyze, and interpret chemical data.</li>
    <li><b>üßÆ Mathematical Modeling:</b> I have used Genome-scale Models (GEMs) to make predictions using cobrapy. I have also developed kinetic metabolic models, using both deterministic (e.g. ODE) and stochastic (e.g. Kinetic Monte Carlo) kinetic modeling approaches.</li>
    <li><b>‚úçüèª Scientific Writing:</b> I clearly communicate my science to the rest of the world or company. My work has been published in <i>Bioinformatics</i>, <i>PLoS Computational Biology</i>, and <i>Nature Biotechnology</i>.</li>
  </ul>

  <!-- Professional Experience -->
  <h1 class="subheading">Professional Experience</h1>
  <b>LanzaTech (Skokie, IL)</b> - Feb 2024 - Now
  <li>Bioinformatics Software Engineer, AI & Computational Biology</li>

  <b>LanzaTech (Skokie, IL)</b> - July 2022 - Jan 2024
  <li>Scientist, AI & Modeling</li>

  <b>Northwestern University (Evanston, IL)</b> - 2016 - 2022
  <li>PhD Candidate in Chemical & Biological Engineering</li>

  <b>LanzaTech (Skokie, IL)</b> - June 2018 - Dec 2018
  <li>Computational Biology Intern</li>

  <b>Wood Laboratory at Ohio State (Columbus, OH)</b> - 2014 - 2016
  <li>Undergraduate Researcher</li>

  <b>Ohio State University (Columbus, OH)</b> - 2013 - 2015
  <li>Undergraduate Teaching Assistant</li>

  <b>DuPont (Circleville, OH)</b> - Summer 2013
  <li>Manufacturing Technology Engineer Intern</li>

  <!-- Education -->
  <h1 class="subheading">Education</h1>
  <b>Northwestern University (Evanston, IL)</b> - Sep 2016 - June 2022
  <li>Ph.D. Chemical & Biological Engineering</li>
  <li>Advisors: Keith Tyo & Linda Broadbelt</li>
  <li>Thesis: <i>Computational Approaches for Enhanced Metabolomics Annotation and DNA-based Biorecording and Data Storage</i></li>

  <b>Ohio State University (Columbus, OH)</b> - Sep 2011 - Dec 2015
  <li>B.S. Chemical & Biomolecular Engineering</li>
  <li>Minors: Biomedical Engineering, German</li>
  <li>GPA: 3.82</li>

  <b>Circleville High School (Circleville, OH)</b> - 2007 - 2011
  <li>Valedictorian (GPA: 4.55)</li>

  <!-- Honors & Awards -->
  <h1 class="subheading">Honors & Awards</h1>
  <li>ARCS Scholar Award (renewed twice) - 2017, 2018, 2019</li>
  <li><b>Biotechnology Training Program Trainee</b> - 2017, 2018</li>
  <li><b>Undergraduate Research Award</b> - 2015</li>
  <li>Honda-OSU Math Medal Award - 2011</li>
  <li>Columbus Dispatch Scholar-Athlete Runner-Up Male</b> - 2011</li>
  <li>National Merit Finalist - 2011</li>
  <li>High School Valedictorian - 2011</li>

  <!-- Teaching Experience -->
  <h1 class="subheading">Teaching Experience</h1>
  <b>Northwestern University (Evanston, IL)</b>
  <li>ChBE 373 - Global Health & Biotechnology (TA) - F2019</li>
  <li>ChBE 477 - Bioseparations (TA) - S2017, S2018, S2019</li>
  
  <b>Ohio State University (Columbus, OH)</b>
  <li>CBE 2420 - Transport Phenomena I - F2013, F2014, F2015</li>

  <!-- Organizations -->
  <h1 class="subheading">Organizations</h1>
  <b>Northwestern University (Evanston, IL)</b>
  <li>Graduate Student Forum - 2017 - 2018</li>

  <b>Ohio State University (Columbus, OH)</b>
  <li>Phi Kappa Tau Fraternity, Gamma Chapter
    <ul style='margin-top: 0px;'>
      <li>Vice President - 2013 - 2015</li>
      <li>Secretary - 2012</li>
      <li>Founding Father - 2012</li>
    </ul>
  </li>

  <!-- Publications -->
  <h1 class="subheading">Publications</h1>
  
  <div style="font-size: 16px;">
    <b>Primary Research</b>
    <li>Shebek K, <b>Strutz J</b>, Broadbelt L, Tyo K (2023)
    Pickaxe: a Python library for the prediction of novel metabolic reactions
    <i>BMC Bioinformatics</i> 24:106</li>

    <li><b>Strutz J</b>, Shebek K, Broadbelt L, Tyo K (2022)
    MINE 2.0: enhanced biochemical coverage for peak identification in untargeted metabolomics.
    <i>Bioinformatics</i>, 38(13):3484-3487.</li>

    <li>Liew EF, Nogle R, Abdalla T, Rasor BJ, Canter C, Jensen RO, Wang L, <b>Strutz J</b>, ..., K√∂pke M (2022)
    Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale.
    <i>Nature Biotechnology</i>, 40:335-344.</li>

    <li>Bhan NJ, Callisto A, <b>Strutz J</b>, Glaser J, Kalhor R, Boyden E, Church G, Kording K, Tyo KEJ (2021)
    Recording Temporal Signals with Minutes Resolution Using Enzymatic DNA Synthesis.
    <i>Journal of the American Chemical Society</i>, 143:16630-16640.</li>
    
    <li>St. John P, <b>Strutz J</b>, Broadbelt LJ, Tyo KEJ, Bomble YJ (2019)
    Bayesian inference of metabolic kinetics from genome-scale multiomics data.
    <i>PLoS Computational Biology</i>, 15(11):e1007424.</li>
    
    <b>Reviews and Opinions</b>
    <li><b>Strutz J</b>, Martin J, Greene J, Broadbelt L, Tyo K (2019)
    Metabolic kinetic modeling provides insight into complex biological questions, but hurdles remain.
    <i>Current Opinion in Biotechnology</i>, 59:24-30.</li>
    
    <b>Book Chapters and Technical Reports (Non-peer Reviewed)</b>
    <li>Jeffryes J, <b>Strutz J</b>, Henry C, Tyo KEJ (2019)
    Metabolic in silico network expansions to predict and exploit enzyme promiscuity.
    <i>Methods in Molecular Biology</i>, 1927:11-21.</li>

    <li>Rosenfeld, D, Liew, F, Kopke, M, Gao, A, Harris, A, Mueller, A, Nagaraju, S, Smith, A, Tran, L, O'Brien, J, Tyo, K, Martin, J, <b>Strutz, J</b>, Metz, J. The Dow Chemical Company (2019)
    Bio-syngas to Fatty Alcohols (C6-14) as a Pathway to Fuels.</li>
  </div>

  <!-- Presentations, Career Panels, & Talks -->
  <h1 class="subheading">Presentations, Panels, & Talks</h1>

  <div style="font-size: 16px;">
    <b>November 28, 2023 - Northwestern University Data Science Nights Career Panel - Virtual</b>
    <li>Participated in a career panel for data science students considering a career in industry</li>

    <b>October 27, 2023 - Northwestern University Center for Synthetic Biology Career Panel - Chicago, IL</b>
    <li>Participated in a career panel for synthetic biology graduate students considering a career in industry</li>

    <b>February 24, 2022 - PhD Student Seminar - Evanston, IL</b>
    <li>‚ÄúExploring the Uncharted Biochemical Universe: Predicting Metabolites across Scales‚Äù</li>
    
    <b>September 10, 2021 - Central US SynBio Workshop - Evanston, IL</b>
    <li>‚ÄúMINE 2.0: Enhanced biochemical coverage for peak identification in untargeted metabolomics‚Äù (poster)</li>

    <b>October 7, 2020 - ARCS Fun at Five - Evanston, IL</b>
    <li>‚ÄúUsing DNA as a medium for recording biological signals‚Äù</li>

    <b>September 10, 2020 - Central US SynBio Workshop - Online</b>
    <li>‚ÄúKinetic Monte Carlo Modeling of ssDNA Extension via Terminal deoxynucleotidyl Transferase‚Äù (poster)</li>

    <b>May 8, 2020 - Virtual Podium 2020 - Online</b>
    <li>‚ÄúMINEs: Open Access Databases of Computationally Predicted Enzyme Products for Untargeted Metabolomics‚Äù</li>

    <b>September 12, 2019 - Central US SynBio Workshop - Madison, WI</b>
    <li>‚ÄúPrediction of Liquid Chromatography Retention Times for Metabolite Identification‚Äù (poster)</li>

    <b>May 21, 2019 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
    <li>‚ÄúDiscovery of Novel Metabolites and Gene Functions in Aromatic-Degrading Organisms (Proposed Work)‚Äù</li>

    <b>October 31, 2018 - AIChE Annual Meeting - Pittsburgh, PA</b>
    <li>‚ÄúAddition of Thermodynamic Constraints to a Scalable Kinetic Modeling Framework‚Äù</li>

    <b>March 4, 2018 - ARCS Illinois Chapter Meet & Greet - Evanston, IL</b>
    <li>‚ÄúImproving the Environmental Sustainability of Biorefineries: Engineering a Bacterium to Eat Plants‚Äù</li>

    <b>June 26, 2017 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
    <li>‚ÄúModeling Catabolism of Lignin-Derived Aromatics in Acinetobacter baylyi ADP1‚Äù</li>
  </div>

</div>

</body>

</html>