---
title: CV
---

<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <title>{{ page.title }}</title>
</head>

<body>
  <!-- Download Link -->
  <!-- <div class="bodyText" style='margin-top: 30px; margin-bottom: 20px;'>
    <a href="/assets/pdfs/cv.pdf"><h2><u>Download CV as PDF</u></h2></a>
  </div> -->

  <!-- CV in HTML -->
  <div class="bodyText">

  <!-- Summary -->
  <h1>Summary</h1>
  <p>
    I am currently working a Scientist at LanzaTech where I develop software, analyze data, and work with experimentalists to engineer microorganisms to produce useful chemicals sustainably.
  </p>
  <p>
    I'm fluent in Python, MongoDB, SQL, and English. I've also written a decent amount of JavaScript, HTML, CSS, Swift, and Japanese. I love to learn new technologies and use those tools to address practical needs.
  </p>
  <p>
    I currently live in Skokie, IL and in my spare time I enjoy hiking, reading, writing, software development, and strength training.
  </p>

  <!-- Skills -->
  <h1>Skills: What can I do?</h1>
  <h3>Scientific Research & Data Science</h3>
  <ul>
    <li><b>Design experiments:</b> I absolutely love working with experimentalists to design experiments and analyses that best answer the most pressing questions.</li>
    <li><b>Perform statistical analyses:</b> I have experience with both frequentist statistics as well as Bayesian statistical modeling.</li>
    <li><b>Build machine learning models:</b> Analyze complex datasets and make predictions. For example, I built a random forest regressor model using scikit-learn to predict chromatographic retention times in order to annotate unknown compounds in a metabolomics dataset.</li>
    <li><b>Use cheminformatics tools:</b> I have used tools like RDKit and mordred to process, analyze, and interpret chemical data.</li>
    <li><b>Model biological systems:</b> I have used Genome-scale Models (GEMs) to make predictions using cobrapy. I have also developed kinetic metabolic models, using both deterministic (e.g. ODE) and stochastic (e.g. Kinetic Monte Carlo) kinetic modeling approaches.</li>
    <li><b>Visualize data:</b> Data visualization is one of my favorite parts of doing science. I am most experienced with python libraries such as matplotlib and seaborn. I have also used tools like streamlit to create interactive dashboards to visualize data in user-defined ways.</li>
    <li><b>Write scientific papers and internal reports:</b> Clearly communicate our science to the rest of the world or company. I have been published in PLoS Computational Biology and Current Opinion in Biotechnology.</li>
    <li><b>Give technical talks and training:</b> Provide expertise in computational biology, data science, and metabolic engineering, which I have 5+ years of experience with.</li>
    </ul>
  <h3>Software Development</h3>
  <ul>
    <li><b>Develop research tools and APIs:</b> For example, I helped develop the <a href="https://minedatabase.ci.northwestern.edu/#/home"><u>MINE Database</u></a>, to unlock new modes of scientific inquiry. I have used flask to build multiple web APIs.</li>
    <li><b>DevOps and Scripting:</b> Create Docker images, deploy to EC2 on AWS using CodeDeploy, set security for incoming/outbound traffic in AWS, bat and bash scripting.</li>
    <li><b>High-Performance Computing:</b> I have used the Slurm Workload Manager to submit jobs on supercomputing clusters. For example, I used the Quest supercomputing cluster at Northwestern University to predict MS2 spectra for millions of compounds in parallel.</li>
    <li><b>Build and query databases:</b> I have experience building and working with MySQL (with sqlalchemy driver) as well as MongoDB (with pymongo driver) databases.</li>
    <li><b>Contribute to frontend development:</b> I'm familiar with vanilla JavaScript as well as basic HTML and CSS. I built this site using Jekyll, HTML, and CSS, for example.</li>
    <li><b>Collaborate using git and GitHub:</b> submit issues, make commits, merge changes, review and submit pull requests, organize issues into projects. I have used git and GitHub for both solo projects as well as collaborative projects.</li>
    <li><b>Communicate and document effectively:</b> I prefer to use numpy-style docstrings in my python code. I also implement best practices, like adding python type annotations and using descriptive variable names. I have used Sphinx to generate API documentation.</li>
    <li><b>Develop iOS applications using SwiftUI:</b> As a side project, I'm currently working on my first iPhone app - a crossword game for Japanese language learners (see blog for details).</li>
  </ul>

  <!-- Professional Experience -->
  <h1>Professional Experience</h1>
  <b>LanzaTech (Skokie, IL)</b> - July 2022 - Now
  <li>Scientist, AI & Modeling</li>

  <b>Northwestern University (Evanston, IL)</b> - 2016 - 2022
  <li>PhD Candidate in Chemical & Biological Engineering</li>

  <b>LanzaTech (Skokie, IL)</b> - June 2018 - Dec. 2018
  <li>Computational Biology Intern</li>

  <b>Wood Laboratory at Ohio State (Columbus, OH)</b> - 2014 - 2016
  <li>Undergraduate Researcher</li>

  <b>Ohio State University (Columbus, OH)</b> - 2013 - 2015
  <li>Undergraduate Teaching Assistant</li>

  <b>DuPont (Circleville, OH)</b> - Summer 2013
  <li>Manufacturing Technology Engineer Intern</li>

  <!-- Education -->
  <h1>Education</h1>
  <b>Northwestern University (Evanston, IL)</b> - Sept. 2016 - June 2022
  <li>Ph.D. Chemical & Biological Engineering</li>
  <li>Advisors: Keith Tyo & Linda Broadbelt</li>
  <li>Thesis: Computational Approaches for Enhanced Metabolomics Annotation and DNA-based Biorecording and Data Storage</li>

  <b>Ohio State University (Columbus, OH)</b> - Sept. 2011 - Dec. 2015
  <li>B.S. Chemical & Biomolecular Engineering</li>
  <li>Minors: Biomedical Engineering, German</li>
  <li>GPA: 3.82</li>

  <!-- Honors & Awards -->
  <h1>Honors & Awards</h1>
  <li>ARCS Scholar Award (renewed twice) - 2017, 2018, 2019</li>
  <li><b>Biotechnology Training Program Trainee</b> - 2017, 2018</li>
  <li><b>Undergraduate Research Award</b> - 2015</li>
  <li>Honda-OSU Math Medal Award - 2011</li>
  <li>Columbus Dispatch Scholar-Athlete Runner-Up Male</b> - 2011</li>
  <li>National Merit Finalist - 2011</li>
  <li>High School Valedictorian - 2011</li>

  <!-- Teaching Experience -->
  <h1>Teaching Experience</h1>
  <b>Northwestern University (Evanston, IL)</b>
  <li>ChBE 373 - Global Health & Biotechnology (TA) - F2019</li>
  <li>ChBE 477 - Bioseparations (TA) - S2017, S2018, S2019</li>
  
  <b>Ohio State University (Columbus, OH)</b>
  <li>CBE 2420 - Transport Phenomena I - F2013, F2014, F2015</li>

  <!-- Organizations -->
  <h1>Organizations</h1>
  <b>Northwestern University (Evanston, IL)</b>
  <li>Graduate Student Forum - 2017 - 2018</li>

  <b>Ohio State University (Columbus, OH)</b>
  <li>Phi Kappa Tau Fraternity, Gamma Chapter
    <ul style='margin-top: 0px;'>
      <li>Vice President - 2013 - 2015</li>
      <li>Secretary - 2012</li>
      <li>Founding Father - 2012</li>
    </ul>
  </li>

  <!-- Publications -->
  <h1>Publications</h1>
  
  <b>Primary Research</b>
  <li><b>Strutz J</b>, Shebek K, Broadbelt L, Tyo K (2022)
  MINE 2.0: enhanced biochemical coverage for peak identification in untargeted metabolomics.
  <i>Bioinformatics</i>, 38(13):3484-3487.</li>

  <li>Liew EF, Nogle R, Abdalla T, Rasor BJ, Canter C, Jensen RO, Wang L, <b>Strutz J</b>, ..., Köpke M (2022)
  Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale.
  <i>Nature Biotechnology</i>, 40:335-344.</li>

  <li>Bhan NJ, Callisto A, <b>Strutz J</b>, Glaser J, Kalhor R, Boyden E, Church G, Kording K, Tyo KEJ (2021)
  Recording Temporal Signals with Minutes Resolution Using Enzymatic DNA Synthesis.
  <i>Journal of the American Chemical Society</i>, 143:16630-16640.</li>
  
  <li>St. John P, <b>Strutz J</b>, Broadbelt LJ, Tyo KEJ, Bomble YJ (2019)
  Bayesian inference of metabolic kinetics from genome-scale multiomics data.
  <i>PLoS Computational Biology</i>, 15(11):e1007424.</li>
  
  <b>Reviews and Opinions</b>
  <li><b>Strutz J</b>, Martin J, Greene J, Broadbelt L, Tyo K (2019)
  Metabolic kinetic modeling provides insight into complex biological questions, but hurdles remain.
  <i>Current Opinion in Biotechnology</i>, 59:24-30.</li>
  
  <b>Book Chapters and Technical Reports (Non-peer Reviewed)</b>
  <li>Jeffryes J, <b>Strutz J</b>, Henry C, Tyo KEJ (2019)
  Metabolic in silico network expansions to predict and exploit enzyme promiscuity.
  <i>Methods in Molecular Biology</i>, 1927:11-21.</li>

  <li>Rosenfeld, D, Liew, F, Kopke, M, Gao, A, Harris, A, Mueller, A, Nagaraju, S, Smith, A, Tran, L, O'Brien, J, Tyo, K, Martin, J, <b>Strutz, J</b>, Metz, J. The Dow Chemical Company (2019)
  Bio-syngas to Fatty Alcohols (C6-14) as a Pathway to Fuels.</li>
  
  <!-- Presentations & Talks -->
  <h1>Presentations & Talks</h1>
  <b>February 24, 2022 - PhD Student Seminar - Evanston, IL</b>
  <li>“Exploring the Uncharted Biochemical Universe: Predicting Metabolites across Scales”</li>
  
  <b>September 10, 2021 - Central US SynBio Workshop - Evanston, IL</b>
  <li>“MINE 2.0: Enhanced biochemical coverage for peak identification in untargeted metabolomics” (poster)</li>

  <b>October 7, 2020 - ARCS Fun at Five - Evanston, IL</b>
  <li>“Using DNA as a medium for recording biological signals”</li>

  <b>September 10, 2020 - Central US SynBio Workshop - Online</b>
  <li>“Kinetic Monte Carlo Modeling of ssDNA Extension via Terminal deoxynucleotidyl Transferase” (poster)</li>

  <b>May 8, 2020 - Virtual Podium 2020 - Online</b>
  <li>“MINEs: Open Access Databases of Computationally Predicted Enzyme Products for Untargeted Metabolomics”</li>

  <b>September 12, 2019 - Central US SynBio Workshop - Madison, WI</b>
  <li>“Prediction of Liquid Chromatography Retention Times for Metabolite Identification” (poster)</li>

  <b>May 21, 2019 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
  <li>“Discovery of Novel Metabolites and Gene Functions in Aromatic-Degrading Organisms (Proposed Work)”</li>

  <b>October 31, 2018 - AIChE Annual Meeting - Pittsburgh, PA</b>
  <li>“Addition of Thermodynamic Constraints to a Scalable Kinetic Modeling Framework”</li>

  <b>March 4, 2018 - ARCS Illinois Chapter Meet & Greet - Evanston, IL</b>
  <li>“Improving the Environmental Sustainability of Biorefineries: Engineering a Bacterium to Eat Plants”</li>

  <b>June 26, 2017 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
  <li>“Modeling Catabolism of Lignin-Derived Aromatics in Acinetobacter baylyi ADP1”</li>

</div>

</body>

</html>